x/axelarnet/client/cli/tx.go

```diff
@@ -76,6 +77,13 @@ func GetCmdConfirmDeposit() *cobra.Command {
 			}
 
 			txID, err := hex.DecodeString(args[0])
+			if err != nil {
+				return err
+			}
+
+			if len(txID) != common.HashLength {
+				return fmt.Errorf("txID should be %d bytes", common.HashLength)
+			}
```

Comment:
this TX hash is for cosmos based chains, not EVM chains. Doesn't cosmos have its own constant for an hash lenght?

Reply:
Couldn't seem to find one in cosmos sdk unless you know where it is?

Reply:
I had this 
```
	if len(m.TxID) != 32 {
		return fmt.Errorf("invalid TxID")
	}
```

Reply:
I think it is preferable to use Haiyi's approach, if there is indeed no constant in cosmos sdk.

Reply:
that's fine, we will remove txid in the next release 😄

Reply:
but it is from a different module/chaintype. I don't really like doing that. But I don't feel super strong about it. I leave it to your judgment.

Reply:
> it's fine to use hashlength, we will remove txid in the next release 😄

Then why even have this check? IIRC, we don't actually use this field